_sass/us-design-standards/lib/bourbon/css3/_image-rendering.scss

```diff
@@ -0,0 +1,14 @@
+@mixin image-rendering ($mode:auto) {
+
+  @if ($mode == crisp-edges) {
+    -ms-interpolation-mode: nearest-neighbor; // IE8+
```

Comment:
Properties should be ordered image-rendering, image-rendering, image-rendering, image-rendering, -ms-interpolation-mode